.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+**/*/.DS_store
```

Comment:
Small thing: remove `.DS_Store`s that have already been committed when this line came in.